source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -357,26 +381,30 @@ struct functor_add_vj {
             //this vehicle journey won't ever circulate
             return false;
         }
-        vj->adapted_validity_pattern = pt_data.get_or_create_validity_pattern(tmp_vp);
+        auto meta_vj_it = pt_data.meta_vj.find(vj_ref.uri);
+        if (meta_vj_it != pt_data.meta_vj.cend()) {
+            push_back_unique(pt_data.meta_vj[vj_ref.uri]->adapted_vj, &vj);
+        }
+        vj.adapted_validity_pattern = pt_data.get_or_create_validity_pattern(tmp_vp);
         vj_ref.adapted_validity_pattern = pt_data.get_or_create_validity_pattern(tmp_ref_vp);
```

Comment:
Ok, looks like a bigger refactor (and probably contained in the one for RT).
So I guess it's gonna stay this way for now :-/